lib/internal/Magento/Framework/Mview/View/Changelog.php

```diff
@@ -6,8 +6,15 @@
 
 namespace Magento\Framework\Mview\View;
 
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\SessionException;
 use Magento\Framework\Phrase;
 
+/**
+ * Class Changelog
+ *
+ * @package Magento\Framework\Mview\View
+ */
```

Comment:
I think this information is redundant.

Could you change with comment about general purposes of this class?